src/main/java/com/gachigang/ontherun/common/exception/UserAlreadyExistException.java

```diff
@@ -0,0 +1,11 @@
+package com.gachigang.ontherun.common.exception;
+
+/**
+ * Exception that is thrown when a user attempts to register, but
+ * the provided user login is already in use.
+ */
```

Comment:
ResponseStatus(HttpStatus.BAD_REQUEST) ?

Reply:
The same